lldb/include/lldb/Core/ModuleSpec.h

```diff
@@ -265,6 +274,9 @@ class ModuleSpec {
   ArchSpec m_arch;
   UUID m_uuid;
   ConstString m_object_name;
+  /// This is set to take advantage of the target's search path and platform's
+  /// locate module callback
+  std::weak_ptr<Target> m_target;
```

Comment:
change the name to `m_target_wp` to indicate a weak pointer. We name shared pointers with a `_sp` suffix and weak pointers with a `_wp`